contents/tutorials/calculating-new-returning-users.md

```diff
@@ -0,0 +1,93 @@
+---
+title: How to calculate new and returning users in PostHog
```

Comment:
```suggestion
title: How to track new and returning users in PostHog
```

People are more likely to search for how to track than how to calculate. Should aid discovery.